src/styles/theme.ts

```diff
@@ -428,10 +429,10 @@ const initTheme = (darkMode: boolean) => {
       },
       MuiBackdrop: {
         styleOverrides: {
-          root: {
+          root: ({ theme }) => ({
             backdropFilter: 'blur(1px)',
-            backgroundColor: 'rgba(228, 232, 241, 0.75)',
-          },
+            backgroundColor: setOpacity(theme.palette.background.main || '', 0.75),
```

Comment:
MUI ships with a util function called `alpha` which we can use: `alpha(theme.palette.background.main, 0.75)`. Do we need the fallback to ''? Afaik `background.main` is defined in both color palettes.